src/resources/Organizations.js

```diff
@@ -1,17 +1,61 @@
-import React from "react";
+import PropTypes from "prop-types";
+import { useState } from "react";
 import "./resources.css";
 
+const fakeData = require("./fakeData");
```

Comment:
Using require() to import  will cause no issue. However, it is not good for readability. For frontend, use import as usual 
`import { organizationsMap as <name_you_prefer> } from "./fakeData"`. We use `require()` for backend because it uses vanilla javascript node.js

Recommend seeing `EventsList.js` file and es6 module export on Google